### PR TITLE
website deployment for the portfolio

### PR DESCRIPTION
**Pull request by Yoz21**
In this task, I deploy my portfolio website using Github pages. I Set the theme Jekyll-theme-merlot for the deployment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Portfolio
 
-This is a desktop version of my portfolio.It is done in the template design I have chosen from figma.
+This is a desktop version of my portfolio.It is done in the template design I have chosen from figma.You can see the online version of my portfolio at https://yoz21.github.io/Portfolio/.
```
Comment (rotimiazeez) [inline README.md]: - [ ] Great job deploying your page💯, I'd suggest you make the link easily accessible by adding it as a link👌.
You can try adding it like 
```
## Live Deploy

[Deploy Link](https://yoz21.github.io/Portfolio/)
```

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Yoz21 👌, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Yoz21) [reply README.md]: thanks

File: README.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 This is a desktop version of my portfolio.It is done in the template design I have chosen from figma.
 
+## Live Deploy
+
+[Deploy Link](https://yoz21.github.io/Portfolio/)
+
```
Comment (AleksandarIvezic) [inline README.md:8]: 👍🏼

**Review by AleksandarIvezic — APPROVED** [review]
## Hi @Yoz21  👋🏼 , 

STATUS: Approved! ✔️ ✔️ ✔️ 

Great work on making the changes requested by a previous reviewer. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

gif

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] You can also consider fixing the repeating background images of your project, that appears on smaller screen size. I believe that you agree that it will look much better that way. 😃 

image

- [ ] Also, I think it would look much better on tablet and mobile screens if your about me boxes are centered or repositioned using some grid layout. 😃 

image

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**